src/convertToObject.js

```diff
@@ -7,6 +7,14 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const mass = sourceString.split(';');
```

Comment:
The variable name `mass` is not very descriptive. Consider renaming it to something more meaningful, like `pairs` or `entries`, to improve code readability.